src/react-components/input/Slider.stories.js

```diff
@@ -13,10 +13,10 @@ const Template = args => (
   </Column>
 );
 
-export const Base = Template.bind({});
-Base.argTypes = {
+export const Controlled = Template.bind({});
```

Comment:
I have not really used storybook much, whats this change about?

Reply:
After learning about the Controlled/Uncontrolled ReactJS components I updated the slider story to show the slider component working as a controller and an uncontrolled component.